client/src/languages/en.js

```diff
@@ -347,7 +347,7 @@ const en = defineMessages({
     },
     signInPageTitle: {
         id: 'sign-in-page.title',
-        defaultMessage: 'Login',
+        defaultMessage: 'LOGIN',
```

Comment:
I think we shouldn't change it to uppercase here. We should convert text to upper case in usage place.